blockmap.js

```diff
@@ -34,13 +34,13 @@ function mergeBlocks(blocks) {
 }
 
 function streamSha256(stream, callback) {
-	const hash = sha256.create();
-	stream.on('data', function(data) {
-		hash.update(data);
-	});
+	const hash = crypto.createHash('sha256');
+	hash.setEncoding('hex');
+	stream.pipe(hash);
 	stream.on('error', callback);
 	stream.on('end', function() {
-		callback(null, hash.hex());
+		hash.end();
+		callback(null, hash.read());
```

Comment:
That was the first version:

```javascript
function streamSha256(stream, callback) {
	const hash = crypto.createHash('sha256');
	stream.on('data', function(data) {
		hash.update(data);
	});
	stream.on('error', callback);
	stream.on('end', function() {
		callback(null, hash.digest('hex'));
	});
}
```
It was amended after [@jhermsmeier 's comment](https://github.com/resin-io-modules/file-disk/pull/2#pullrequestreview-46319109)
Both versions work.
I have no preference.